index.html

```diff
@@ -1 +1,342 @@
+											</div>
+								        </li>
+								        <li class="mb-2">
+								            <div class="resume-skill-name">Azure Cloud</div>
+									        <div class="progress resume-progress">
+											    <div class="progress-bar theme-progress-bar-dark" role="progressbar" style="width: 86%" aria-valuenow="25" aria-valuemin="0" aria-valuemax="100"></div>
+											</div>
+								        </li>
+								        <li class="mb-2">
+								            <div class="resume-skill-name">Redux</div>
+									        <div class="progress resume-progress">
+											    <div class="progress-bar theme-progress-bar-dark" role="progressbar" style="width: 82%" aria-valuenow="25" aria-valuemin="0" aria-valuemax="100"></div>
+											</div>
+								        </li>
+							        </ul>
+						        </div><!--//resume-skill-item-->
+						        
+						        <div class="resume-skill-item">
+						            <h4 class="resume-skills-cat font-weight-bold">Others</h4>
+						            <ul class="list-inline">
+							            <li class="list-inline-item"><span class="badge badge-light">DevOps</span></li>
+							            <li class="list-inline-item"><span class="badge badge-light">Code Review</span></li>
+							            <li class="list-inline-item"><span class="badge badge-light">Git</span></li>
+							            <li class="list-inline-item"><span class="badge badge-light">Unit Testing</span></li>
+							            <li class="list-inline-item"><span class="badge badge-light">Wireframing</span></li>
+							            <li class="list-inline-item"><span class="badge badge-light">Sketch</span></li>
+							            <li class="list-inline-item"><span class="badge badge-light">Gitflow</span></li>
+							            <li class="list-inline-item"><span class="badge badge-light">MVC</span></li>
+							            <li class="list-inline-item"><span class="badge badge-light">NPM</span></li>
+						            </ul>
+						        </div><!--//resume-skill-item-->
+						    </div><!--resume-section-content-->
+					    </section><!--//skills-section-->
+					    <section class="resume-section education-section mb-5">
+						    <h2 class="resume-section-title text-uppercase font-weight-bold pb-3 mb-3">Education</h2>
+						    <div class="resume-section-content">
+							    <ul class="list-unstyled">
+								    <li class="mb-2">
+								        <div class="resume-degree font-weight-bold">B.Tech Electronics and Communication</div>
+								        <div class="resume-degree-org">Cochin University of Science and Technology</div>
+								        <div class="resume-degree-time">2006-Pursuing</div>
+								    </li>
+								    <li>
+								        <div class="resume-degree font-weight-bold">ASP.NET Full Stack Development Course</div>
+								        <div class="resume-degree-org">STEPS, Kochi</div>
```

Comment:
change STEPS

Reply:
STEPS is the name of the institution, why should it be removed?